build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,68 @@
+""" Script to create Sagemaker TensorFlow Docker images
+
+    Usage:
+        python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
+"""
+import argparse
+import glob
+import os
+import shutil
+import subprocess
+import sys
+
+## GLOBALS
+DOCKER = 'docker' # Stores which type of DOCKER to use, nvidia-docker or docker
+PATH_TO_SCRIPT = ''  # Stores route the user takes to build scripts directory
+
+def create_docker_image(optbin_link, processor, framework_version, python_version):
+    """ Function builds a docker image with the TF optimized binary
+
+    Args:
+        optbin_link (str): link to where the optimized binary is
+        processor (str): gpu or cpu
+        framework_version (str): tensorflow version i.e 1.6.0
+        python_version (str): (i.e. 3.6.5 or 2.7.4)
+    """
+    # Initialize commonly used variables
+    py_v = 'py{}'.format(python_version.split('.')[0]) # i.e. py2
+
+    # Get optimized binary - and put in final docker image repo
+    print('Getting optimized binary...')
+    optbin_filename = 'tensorflow-{}-cp27-cp27mu-manylinux1_x86_64.whl'.format(framework_version)
+    with open('{}/../docker/{}/final/{}/{}'.format(PATH_TO_SCRIPT, framework_version, py_v, optbin_filename), 'wb') as optbin_file:
+        subprocess.call(['curl', optbin_link], stdout=optbin_file)
```

Comment:
`optbin_link` needs to be an HTTP(S) URL, right? I think this script should be agnostic about the protocol. I think it should accept local files, for example, file:///my-special-tensorflow.tar.gz. Instead of `optbin_link`, I suggest --path-to-tensorflow-binaries, which can install from http(s):// or file://.

Reply:
Ok here is my proposed fix:
```python
optbin_filename = 'tensorflow-{}-{}-binary.whl'.format(framework_version, processor)
if os.path.isfile(optbin_path):
    shutil.copyfile(optbin_filename, '{}/../docker/{}/final/{}/{}'.format(PATH_TO_SCRIPT, framework_version, py_v, output_filename))
else:
    with open('{}/../docker/{}/final/{}/{}'.format(PATH_TO_SCRIPT, framework_version, py_v, optbin_filename), 'wb') as optbin_file:
        subprocess.call(['curl', optbin_link], stdout=optbin_file)

```